services/website-api/src/app.ts

```diff
@@ -132,11 +143,11 @@ const bootServer = async () => {
 	});
 
 	app.get(
-		'/api/website-api/openapi.yaml',
+		'/website-api/openapi.yaml',
 		cors<express.Request>(),
 		basicAuth({
 			users: {
-				docs: 'needAllInfo',
+				hurrdurrpurr: 'camelCaseWithCamelToe',
```

Comment:
hardcoding super secret passwords?

Reply:
that needs to be replaced with process variables, didn't do that yet

Reply:
now we can never use those perfect passwords again, good job.